tasks-dev/docs.js

```diff
@@ -55,5 +55,7 @@ gulp.task('docs:deploy', callback => {
  *
  */
 gulp.task('docs:release', () => gulp.src(`${pathBuilder.docsDistDir}/**/*`)
-    .pipe(ghPages())
+    .pipe(ghPages({
+        message: 'Minor : Documentation Update ' + new Date().toISOString()
```

Comment:
The lint task is failing; this needs to be 

```js
message: `Minor : Documentation Update ${new Date().toISOString()}`
```

Reply:
Yep – that’ll teach me for trying to rush it in ;)